README.md

```diff
@@ -42,7 +42,7 @@ Framework for test verification of state of cluster based on EVM
 Using SSH:
 
 ```plain
-git clone git@github.com:NilFoundation/zkEVM-framework.git
+git clone https://github.com/NilFoundation/zkEVM-framework.git
```

Comment:
`git clone git@github.com:NilFoundation/zkEVM-framework.git` is not link, it's command line for clone via github API. Works for me.

Reply:
yes i know :) mean it wasnt work for me

Reply:
it's required authentication. Your modification just looks like command on the 51th line

Reply:
it's required authentication. Your modification just looks like command on the 51th line